.github/workflows/run_tests.yml

```diff
@@ -25,51 +25,43 @@ jobs:
         with:
           path: ${{ env.pythonLocation }}
           key: test-${{ runner.os }}-${{ env.pythonLocation }}-${{ hashFiles('**/poetry.lock') }}
-      - name: Install dependencies
-        run: pip install . black mypy prospector pytest types-requests
-      - name: lint with black
-        run: black --check .
-      - name: lint with prospector
-        run: make lint
-      - name: lint with mypy
-        run: make mypy
-  test:
-    needs: lint
-    name: test
+      - name: Install (dev) dependencies
+        run: python -mpip install -r requirements/requirements-dev.txt
```

Comment:
Should this be `-m pip`

Reply:
You mean `-m pip` vs `-mpip`? They are the same thing.

Reply:
Fixed: https://github.com/oceanmodeling/searvey/pull/25/commits/e28e87e1c64399f47fd2669aa51cea5008b2d7d5